drivers/virtualbox/virtualbox.go

```diff
@@ -62,17 +65,24 @@ type Driver struct {
 // NewDriver creates a new VirtualBox driver with default settings.
 func NewDriver(hostName, storePath string) *Driver {
 	return &Driver{
-		VBoxManager: NewVBoxManager(),
-		BaseDriver: &drivers.BaseDriver{
-			MachineName: hostName,
-			StorePath:   storePath,
-		},
+		VBoxManager:         NewVBoxManager(),
```

Comment:
Ah, OK, I think we invoke the `NewDriver` function anyway when we `RegisterDriver`, so these fields will be populated.   Nice